auto_round/compressors/base.py

```diff
@@ -2652,7 +2690,10 @@ def quantize_block(
             "DiffusionCompressor",
             "MLLMCompressor",
         ], f"Currently, {self.__class__.__name__} does not support support quantize block with this function."
-        input_ids, input_others = normalize_input(inputs)
+        self.normalize_decoding_layer_inputs_(inputs)
+        block_inputs = self.inputs[self.quant_block_list[0][0]]
+        decoding_layer_first_input_name = "hidden_states"
```

Comment:
does the origin code have such assumption? Is there any case that the first_input_name is not `hidden_states`